liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql

```diff
@@ -610,7 +612,46 @@ BEGIN
                                                )
                                            FOR json path,INCLUDE_NULL_VALUES
                                        ) AS notification_history
-                                  ) AS notification_history
+                                  ) AS notification_history,
+                                  (select
+                                        (
+	                                        select
+		                                        phcase.group_case_cnt                                           as investigation_count,
+		                                        round(COALESCE (gcs.group_case_cnt, phcase.group_case_cnt),0)   as case_count,
```

Comment:
So we pick the value the count that exists between SummaryFormQ or the PHC's group_case_count at the core of the logic?

Reply:
yes, that is correct.